### DISCUSSION TITLE
Is there a contact point for privately reporting potential application issues?

### DISCUSSION THREAD

**Discussion by trevor-vaughan** [NUWCDIVNPT/stig-manager#965] [General]
I'd like to run things by the team prior to posting to the public discussion forums.

**Comment by cd-rite** [comment]
Hi @trevor-vaughan Our [Security Policy](https://github.com/NUWCDIVNPT/stig-manager/security/policy) has the contact email.

**Reply by trevor-vaughan** [reply]
And somehow I managed to look in every file except for the most obvious one 🤦
